lib/sentry/sanitizer/cleaner.rb

```diff
@@ -64,12 +64,12 @@ def sanitize_hash!(event)
       end
 
       def sanitize_breadcrumb!(breadcrumb)
-        return unless breadcrumbs_json_data_fields
```

Comment:
> [!NOTE]
> Removed this as otherwise Rubocop flagged the method with a `Metrics/CyclomaticComplexity` error. As you are setting `@breadcrumbs_json_data_fields = config.breadcrumbs.json_data_fields || []` in the initialiser I though this was ok.